### ISSUE TITLE
Enabling Hyper-V windows feature crashes WSL

### ISSUE THREAD

**Issue by sajithneyo** [microsoft/WSL#5884]
# Environment

```none
Windows build number: 10.0.19041.0 Microsoft Windows NT 10.0.19041.0
Your Distribution version: Ubuntu 18.04 LTS
Whether the issue is on WSL 2 and/or WSL 1: WSL 2
```

# Steps to reproduce

Enable WSL in windows features
Enable Hyper-V in windows features

#  Expected behavior

WSL2 related services to work normally.

# Actual behavior

WSL failes with 
```
Error: 0x80041002

[process exited with code 4294967295]
```

I struggled with #5760 for a month. Yesterday a new windows update came. Then I dissables WSL and Hyper-V from windows features. Then enabled WSL and installed Ubuntu on WSL2 (It works fine). Then I went on with turning Hyper-V (Needed for docker) and did a restart. Now WSL does not works. Throws above error. 

How can I fix this?

**Comment by sajithneyo** [comment]
@therealkenc  I opened #5760 and this issue is not a duplicate of it.

**Comment by therealkenc** [comment]
That you were the OP on #5760 was noted. For what it is worth, #5760 survived a dupe to #5329 at the time it was posted. The submission does not provide any information that is actionable in addition to `Error: 0x80041002`. Your post from two days ago asked "any update on _this_". If you think there is even a snowballs chance that the underlying cause of the new `0x80041002` is different than the old `0x80041002`, please do collect new logs and post the backlink.

**Comment by sajithneyo** [comment]
@therealkenc I have no idea about the internals but last time I could not get WSL2 working anyhow. This whatever the bug, affected me almost for a month. I did upload logs for 5760 in feedback hub. But what happened today is a new scenario following the update I recieved yesterday. WSL2 works but when enabled Hyper-V it crashed with the noted error.

However as I had enough of this, I reinstalled windows. The ISO I got had OS Build 19041.450 and surprisingly again WSL2 started to crash even though it's a fresh install of windows. Then I did an update to the latest (OS Build 19041.508). Now, WSL2 and Hyper-V working perfectly. So I'm closing this issue and 5760.
